src/angular/select/select.ts

```diff
@@ -684,6 +693,71 @@ export class SbbSelect
     }
   }
 
+  /**
+   * Track which modal we have modified the `aria-owns` attribute of. When the combobox trigger is
+   * inside an aria-modal, we apply aria-owns to the parent modal with the `id` of the options
+   * panel. Track the modal we have changed so we can undo the changes on destroy.
+   */
+  private _trackedModal: Element | null = null;
+
+  /**
+   * If the autocomplete trigger is inside of an `aria-modal` element, connect
```

Comment:
Nitpicke; Documentation seems to refer to autocomplete. It should probably be select?